controllers/component_build_controller_pac.go

```diff
@@ -902,3 +1164,64 @@ func retrievePipelineSpec(bundleUri, pipelineName string) (*tektonapi.PipelineSp
 	pipelineSpec := pipelineSpecObj.PipelineSpec()
 	return &pipelineSpec, nil
 }
+
+// updateIncoming updates incomings in repository, adds new incoming for provided branch with incoming secret
+// if repository contains multiple incoming entries, it will merge them to one, and combine Targets and add new secret
+// if repository contains one incoming entry, it will add new target and add new secret
```

Comment:
> and add new secret

Do you mean adding of a new token into existing secret or creation of a new k8s secret? Please be more clear in the comment.

Reply:
changed